R/linters.R

```diff
@@ -0,0 +1,2 @@
+#' @import box.linters
```

Comment:
Please add a comment with the reason for adding this, so a person without context might quickly understand it.